src/main/java/pariveda/osdu/OsduBaseClient.java

```diff
@@ -9,16 +11,36 @@
 	OkHttpClient client;
 	MediaType mediaType;
 	
-	OsduBaseClient(){
-		System.out.println("url, dataparition, contenttype are required");
+	public OsduBaseClient(String url, String dataPartition, String contentType){
```

Comment:
Do we need to pass through content type? Isn't that a known constant of `application/json` for OSDU?

Reply:
this was preserved from Johanna's impl. should be an easy refactor to embed the content type as a final value if this is the case, but wanted to confirm before baking the change into this PR